README.md

```diff
@@ -1,27 +1,28 @@
 # txn-spam 
-
-This is a stress test script you can run against EVM-compatible nodes (geth, polygon-sdk, parity). It will spam various kinds of transactions from multiple base mnemonics and all their child accounts.
+This is a stress test script you can run against borv3. It will spam various kinds of transactions from multiple base mnemonics and all their child accounts.
 
 Ensure you have at least 12 cores if you want to spam from more than 3-4 base mnemonics at once. Ideally 16 cores, otherwise your machine will be the limiting factor and you might not spam as many transactions as you would hope for. 
 
 
 # Setup
 
-Build image if not already:
-
-`docker build . -t txn-spam`
-
-Deploy `Greeter.json` onto your chain and set that address in `spamContractAddress` in `index.ts`
-
-Make a file `mnemonics` in this repo and populate it with 12 word mnemonics, one for each line. Fund the base accounts for each mnemonic with a bunch of ETH. You can use https://iancoleman.io/bip39/ to generate mnemonics and derive the base account. 
+[Optional]:
+    - Make a file `mnemonics` in this repo and populate it with 12 word mnemonics, one for each line. Fund the base accounts for each mnemonic  with a bunch of ETH. You can use https://iancoleman.io/bip39/ to generate mnemonics and derive the base account. 
 
-The script will top up the child accounts to 10 ETH if they are under it, so make sure the base accounts have at least `10 ETH * RANGE` (see below) on your first run.
+All the actions are done with three bash scripts `init.sh`, `start.sh` and `stop.sh`:
+  - `init.sh` - initializes and/or starts bor instances before we can do actual spamming
+      Execute `$ ./init.sh` to see all the commands and options from this init tool
+      Basic/necessary commands are:
+        `$ ./init.sh setup-bor <bor-path>` initialize account and genesis (by default) into `~/borv3/test-dir-`
```

Comment:
Do not call it `bor` but `v3`. That is the placeholder name.

Reply:
fixed